package.json

```diff
@@ -12,6 +12,7 @@
   "scripts": {
     "clean": "rm -rf dist",
     "build": "tsc -p .",
+    "prepublishOnly": "npm run build",
```

Comment:
`dist` is present in `.gitignore`, so it won't be pushed to NPM.

How we have this in `create-nightwatch` is that we build in `lib` directory for development run and `dist` directory for production run.

Reply:
I checked it, and publish would work, as I had added `files` in `package.json`, so even though `dist` is added in `gitignore`, files will still get published